js/viz/core/base_widget.js

```diff
@@ -112,6 +113,14 @@ function callForEach(functions) {
     functions.forEach(c => c());
 }
 
+function floorDimensionsValuesInCanvas(canvas) {
```

Comment:
Isn't it better to name it like this?
```suggestion
function floorCanvasDimensions(canvas) {
```

Reply:
I agree